tests/suite/layout/place.typ

```diff
@@ -121,6 +121,39 @@ Second
   image("/assets/images/diagram.svg", width: 80%),
 )
 
+--- place-float-flush ---
+#set page(height: 150pt, width: 150pt)
+
+#let floater = place(auto, float: true, rect(width: 100%, height: 90pt, text(size: 24pt)[I float!]))
+
+Some introductory text.
+
+#floater #floater #floater #floater
+
+Some additional text.
+
+#place.flush()
+
+Some conclusive text. // Should appear after all the floating figures
+
+--- place-figure-flush ---
```

Comment:
I think this test is a bit redundant. It makes sense to have a test that it works for figures, but it doesn't need to have all the complexity of the previous test. A trimmed-down version with a single floater would likely suffice. Could you also use some other figure content than the SVG. While it's used in some existing figure tests, I try to not use it as much anymore to keep the size of the reference images down.

Reply:
OK, I’ve trimmed this test down.